cg/constants/scout.py

```diff
@@ -148,9 +139,11 @@ class ScoutUploadKey(StrEnum):
 NALLO_SAMPLE_TAGS: dict[str, set[str]] = dict(
     alignment_file={"alignment_haplotags"},
```

Comment:
Don't forget the assembly bam; cf https://github.com/Clinical-Genomics/scout/issues/4982